test/e2e/framework.go

```diff
@@ -52,7 +52,7 @@ import (
 )
 
 const (
-	defaultTimeout = 90 * time.Second
+	defaultTimeout = 120 * time.Second
```

Comment:
this is starting to be a pretty large timeout: what can take up to 2 minutes?

Reply:
forgot to change this value back, this value does not need to be changed.

Reply:
Enabled coverage for the new case TestAntctlProxy.